Misc/NEWS.d/next/Library/2020-02-11-13-01-38.bpo-38691.oND8Sk.rst

```diff
@@ -0,0 +1 @@
+Updated :meth:`importlib._bootstrap_external._make_relax_case` to ignore :envvar:`PYTHONCASEOK` when -E or -I flags are used.
```

Comment:
Please avoid documenting private modules/functions in public Changelog entries. I propose:

```
The :mod:`importlib` module now ignore the :envvar:`PYTHONCASEOK`
environment variable when :option:`-E` or :option:`-I` command line option is used.
```

Please document also this incompatible change at:

* https://docs.python.org/dev/library/functions.html#__import__
* https://docs.python.org/dev/whatsnew/3.9.html#changes-in-the-python-api